odbc/src/api/definitions.rs

```diff
@@ -622,11 +622,15 @@ pub const SQL_CVT_GUID: u32 = 0x01000000;
 pub const MONGO_CAST_SUPPORT: u32 = SQL_CVT_CHAR
     | SQL_CVT_NUMERIC
     | SQL_CVT_INTEGER
```

Comment:
We needed to report that we support conversions to BIGINT, WCHAR, WVARCHAR, and WLONGVARCHAR